project_template/_base/project_name/apps/common/utils/redis.py

```diff
@@ -0,0 +1,35 @@
+import functools
+
+from django.conf import settings
+
+import redis
+
+
+@functools.lru_cache(maxsize=128)
+def redis_client():  # pragma: no cover
+    """
+    Redis client wrapped into LRU cache.
+
+    Example:
+    >>> redis_client().set('foo', 'bar')
+    >>> redis_client().get('foo')
+    """
+    return redis.StrictRedis.from_url(settings.REDIS_URL)
+
+
+def redis_key(prefix, *args, delimiter=':') -> str:
```

Comment:
Yes, you right - it's very simple. But I guess that there are many people in company that haven't used Redis a lot (I mean directly - not as a Celery broker). And this small snippet will show them a good practice for naming keys in Redis using namespaces (instead of putting everything in root of DB by uuids and then receive a collision).